### PR TITLE
Updates dependencies to support PHP 8

### LINKED ISSUE #3: PHP 8.0 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

To be prepared for the december release of PHP 8.0, this repository has some additional TODOs to be tested against the new major version.

In order to make this repository compatible, one has to follow these steps:

- [ ] Modify `composer.json` to provide support for PHP 8.0 by adding the constraint `~8.0.0`
- [ ] Modify `composer.json` to drop support for PHP less than 7.3
- [ ] Modify `composer.json` to implement phpunit 9.3 which supports PHP 7.3+
- [ ] Modify `.travis.yml` to ignore platform requirements when installing composer dependencies (simply add `--ignore-platform-reqs` to `COMPOSER_ARGS` env variable)
- [ ] Modify `.travis.yml` to add PHP 8.0 to the matrix (*NOTE*: Do not allow failures as PHP 8.0 has a feature freeze since 2020-08-04!)
- [ ] Modify source code in case there are incompatibilities with PHP 8.0

### PR DESCRIPTION
**Pull request by driehle**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Fixes #3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,20 +1,24 @@
 {
     "name": "laminas/laminas-mvc-plugins",
-    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
     "type": "metapackage",
-    "license": "BSD-3-Clause",
+    "description": "Collection of all stable laminas-mvc plugins served via separate packages.",
```
Comment (froschdesign) [inline composer.json]: We do not want another normalization because it was already done during the migration from ZF.

File: composer.json [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
     "homepage": "https://laminas.dev",
-    "support": {
-        "issues": "https://github.com/laminas/laminas-mvc-plugins/issues",
-        "source": "https://github.com/laminas/laminas-mvc-plugins",
-        "rss": "https://github.com/laminas/laminas-mvc-plugins/releases.atom",
-        "chat": "https://laminas.dev/chat",
-        "forum": "https://discourse.laminas.dev"
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": "^7.3 || ~8.0.0",
```
Comment (froschdesign) [inline composer.json]: Can you check if we can drop the PHP version constraint? The underlying packages already define the requirements for PHP. Thanks in advance!

File: composer.json [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
+        "laminas/laminas-mvc-plugin-fileprg": "^1.2",
+        "laminas/laminas-mvc-plugin-flashmessenger": "^1.3",
+        "laminas/laminas-mvc-plugin-identity": "^1.2",
+        "laminas/laminas-mvc-plugin-prg": "^1.3",
+        "laminas/laminas-zendframework-bridge": "^1.0"
```
Comment (froschdesign) [inline composer.json]: Also for laminas-zendframework-bridge.

**Review by froschdesign — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (driehle) [reply composer.json]: Sure, that will work.

File: composer.json (hunk 3 above)
Comment (driehle) [reply composer.json]: As there is no code in this repo, dropping the bridge should be finde as well.

Comment (driehle) [reply composer.json]:
```
php8.0 composer.phar why laminas/laminas-zendframework-bridge
[...]
laminas/laminas-mvc-plugin-fileprg         1.2.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-flashmessenger  1.3.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-identity        1.2.0   requires  laminas/laminas-zendframework-bridge (^1.0)
laminas/laminas-mvc-plugin-prg             1.3.0   requires  laminas/laminas-zendframework-bridge (^1.0)
[...]
```

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@driehle 
Thank you for your time and this contribution! 👍

**Pull request merged**